### PR TITLE
fireBase authorisation file

### PR DESCRIPTION
**Pull request by YamanSammani**
This PR is for Firebasebase authorisation file.

### CODE REVIEW

File: src/auth/authContext.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+  // A unique user ID, assigned to the requesting user, we're calling/maticing this ID from our server.
+  let firestoreResult = await firestore.doc("users/" + auth.uid).get();
+  if (!firestoreResult.data()) {
+    await firestore.collection("users").doc(auth.uid).set({
+      uid: auth.uid,
+      fullname: auth.displayName,
+      age: null,
+      userExperiencePoints:0
+
```
Comment (ssinikoglu) [inline src/auth/authContext.jsx]: You may delete extra line here.

File: src/auth/authContext.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    firestoreResult = await firestore.doc("users/" + auth.uid).get();
+  }
+  const user = {
+    isLoggedin: true,
+    uid: auth.uid,
+    email: auth.email,
+    fullname: firestoreResult.data().fullname ?? auth.displayName,
+    age: firestoreResult.data().age ?? null,
+    userExperiencePoints:firestoreResult.data().userExperiencePoints ?? 0
+
```
Comment (ssinikoglu) [inline src/auth/authContext.jsx]: You may delete extra line here.

**Review by ssinikoglu — APPROVED** [review]
I added two minor comments, other than everything seems alright.

**Review by louisrli — COMMENTED** [review]
it looks good btu theres a random file called App.css. do `git rm src/App.css`

File: src/auth/authContext.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    return setUser(null);
+  }
+  // A unique user ID, assigned to the requesting user, we're calling/maticing this ID from our server.
+  let firestoreResult = await firestore.doc("users/" + auth.uid).get();
+  if (!firestoreResult.data()) {
+    await firestore.collection("users").doc(auth.uid).set({
+      uid: auth.uid,
+      fullname: auth.displayName,
+      age: null,
+      userExperiencePoints:0
```
Comment (louisrli) [inline src/auth/authContext.jsx]: space after :. can you run yarn run fix

**Comment by YamanSammani** [comment]
> it looks good btu theres a random file called App.css. do `git rm src/App.css`

Hi @louisrli, I can't see this file in my code and I run the git rm src/App.css and give me : did not match any files. I will commit it now, could you kindly take a look. Thanks.

**Comment by louisrli** [comment]
please check what directory you're running it in. 

or try one of these https://stackoverflow.com/questions/12481639/remove-files-from-git-commit

**Review by ibrahimaljarad — APPROVED** [review]
there is an extra file as louis mentioned if you want to see it checkout to the branch that you created it for this pr, other than this everything is fine.

**Pull request closed without merging**